packages/iris-grid/src/IrisGridTableModelTemplate.ts

```diff
@@ -56,6 +56,8 @@ const log = Log.module('IrisGridTableModel');
 const SET_VIEWPORT_THROTTLE = 150;
 const APPLY_VIEWPORT_THROTTLE = 0;
 
+const EMPTY_ARRAY: unknown[] = [];
```

Comment:
Just making a note that I'll replace this in my readonly change: https://github.com/deephaven/web-client-ui/pull/986